app/services/worldpay-details.service.js

```diff
@@ -0,0 +1,29 @@
+const { ConnectorClient } = require('./clients/connector.client')
+const logger = require('../utils/logger')(__filename)
+
+const connectorClient = new ConnectorClient(process.env.CONNECTOR_URL)
+
+/**
+ *
+ * @param {String} serviceExternalId
+ * @param {String} accountType
+ * @param {WorldpayCredential} credential
+ * @returns {Promise}
+ */
+async function checkCredential (serviceExternalId, accountType, credential) {
+  const credentialCheck = await connectorClient.postCheckWorldpayCredentialByServiceExternalIdAndAccountType(
+    serviceExternalId,
+    accountType,
+    credential.toJson()
+  )
+  if (credentialCheck.result !== 'valid') {
+    logger.warn('Provided credentials failed validation with Worldpay')
+    throw new Error('Invalid credentials')
+  }
+
+  logger.info('Successfully validated credentials with Worldpay')
```

Comment:
this isn't that useful without some additional clarifying information, we might want to also log the service external id